lib/src/main/java/com/what3words/androidwrapper/datasource/text/api/error/BadBoundingBoxError.kt

```diff
@@ -0,0 +1,15 @@
+package com.what3words.androidwrapper.datasource.text.api.error
+
+/**
+ * Represents an error that occurs when an invalid bounding box is used to make a what3words request.
+ *
+ * This error typically indicates that the specified bounding box is not valid or does not conform to the required format
+ * for the request. Instances of this error class provide information about the error through the error code and message.
+ *
+ * @param code The error code associated with the BadBoundingBoxError.
+ * @param message A descriptive message detailing the nature of the error.
+ */
+class BadBoundingBoxError(code: String, message: String) : W3WApiError(
```

Comment:
Should we public the code and message properties (eg. val code: String, override val message)? It helps the upper layers handle the error code and message more flexible

Reply:
thanks for pointing that out mate, I'll revise the errors to make it easier for the clients to access the code and message directly